recipes-sota/ostree/ostree_git.bb

```diff
@@ -73,17 +76,24 @@ do_install_append_class-native() {
 
 
 FILES_${PN} += " \
-    ${@'${systemd_unitdir}/system/' if d.getVar('SYSTEMD_REQUIRED', True) else ''} \
-    ${@'${libdir}/dracut/modules.d/98ostree/module-setup.sh' if d.getVar('SYSTEMD_REQUIRED', True) else ''} \
-    ${datadir}/gir-1.0 \
-    ${datadir}/gir-1.0/OSTree-1.0.gir \
-    ${libdir}/girepository-1.0 \
-    ${libdir}/girepository-1.0/OSTree-1.0.typelib \
+    {bindir} \
+    ${sysconfdir}/ostree \
+    ${datadir}/ostree \
+    ${libdir}/*.so.* \
+    ${libdir}/ostree/ostree-grub-generator \
+    ${libdir}/ostree/ostree-remount \
+    ${libdir}/girepository-1.0/* \
+    ${@bb.utils.contains('DISTRO_FEATURES','systemd','${libdir}/tmpfiles.d', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES','systemd','${systemd_unitdir}/system-generators', '', d)} \
 "
 
-PACKAGES =+ "${PN}-switchroot"
+PACKAGES =+ " \
+    ${PN}-switchroot \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'ostree-dracut', '', d)} \
+"
 
 FILES_${PN}-switchroot = "${libdir}/ostree/ostree-prepare-root"
 RDEPENDS_${PN}-switchroot = ""
 DEPENDS_remove_class-native = "systemd-native"
-
+FILES_${PN}-dev += " ${datadir}/gir-1.0"
+FILES_${PN}-dracut = "${sysconfdir}/dracut.conf.d ${libdir}/dracut"
```

Comment:
There was a bunch of cleanup done of the ostree recipe over the past few months. It looks like a lot of it is here, but I still see substantial differences between the rocko version and this one. Is that necessary due to limitations in morty, or can we get them closer in sync?